packages/boltz-swaps/tests/submarineSign.spec.ts

```
@@ -0,0 +1,187 @@
+    asset: "BTC" as const,
+    swapTree: {} as never,
+    claimPublicKey: CLAIM_PUBLIC_KEY,
+    refundKeys: {
+        privateKey: new Uint8Array(32).fill(1),
+        publicKey: new Uint8Array(33).fill(2),
+    },
+    invoice: "lnbcrt1example",
+});
+
+beforeEach(() => {
+    Object.values(mocks).forEach((m) => m.mockReset());
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)